starknet_py/net/account/account.py

```diff
@@ -359,6 +360,60 @@ async def verify_message(self, typed_data: TypedData, signature: List[int]) -> b
         message_hash = typed_data_dataclass.message_hash(account_address=self.address)
         return await self._verify_message_hash(message_hash, signature)
 
+    @staticmethod
+    async def deploy_account(
+        *,
+        address: int,
+        class_hash: int,
+        salt: int,
+        key_pair: KeyPair,
+        client: Client,
+        chain: StarknetChainId,
+        max_fee: Optional[int] = None,
+        auto_estimate: bool = False,
```

Comment:
I saw the @drknzz comment, but I'm unsure if we should use `auto_estimate` here. It is not a good idea to use it while deploying an account. Also maybe `AccountClient.sign_deploy_account_transaction` shouldn't have an `auto_estimate` argument too?

Reply:
Why isn't this a good idea to use it for deploying your account? I would say sending invokes with this flag is more dangerous.
At the end of the day you can't lose more than what you sent to your account.